lib/afterpay/item.rb

```diff
@@ -6,18 +6,16 @@ module Afterpay
   class Item
     attr_accessor :name, :sku, :quantity, :page_url, :image_url, :price, :categories, :estimated_shipment_date
 
-    # rubocop:disable Metrics/ParameterLists
-    def initialize(name:, price:, page_url:, image_url:, categories:, estimated_shipment_date:, sku: nil, quantity: 1)
-      @name = name
-      @sku = sku
-      @quantity = quantity
-      @price = price
-      @page_url = page_url
-      @image_url = image_url
-      @categories = categories
-      @estimated_shipment_date = estimated_shipment_date
+    def initialize(attributes = {})
+      @name = attributes[:name]
+      @sku = attributes[:sku] || ""
+      @quantity = attributes[:quantity]
```

Comment:
Acc to the API doc the default quantity is 1, shouldn't we be setting that here?

Reply:
@siddyg there is no default value will update if you can point to same in doc.

Reply:
NVM can't find it, I was mistaken